internal/cmd/load-balancer/observability-credentials/add/add_test.go

```diff
@@ -193,3 +192,38 @@ func TestBuildRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestOutputResult(t *testing.T) {
+	type args struct {
+		outputFormat string
+		projectLabel string
+		resp         *loadbalancer.CreateCredentialsResponse
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name:    "empty",
+			args:    args{},
+			wantErr: true,
+		},
+		{
+			name: "only credentials response as argument",
+			args: args{
+				resp: &loadbalancer.CreateCredentialsResponse{Credential: &loadbalancer.CredentialsResponse{}},
```

Comment:
```suggestion
				resp: &loadbalancer.CreateCredentialsResponse{},
```

Maybe add a test case which also covers this case